lib/Helper.ts

```diff
@@ -2,7 +2,8 @@ import { Base64 } from '../enum/Dialogflow';
 
 export const getError = (error: any) => {
     if (typeof error === 'object') {
-        return JSON.stringify(error);
+        const errorObject = Object.getOwnPropertyNames(error).reduce((acc, key) => { acc[key] = error[key]; return acc; }, {});
```

Comment:
For now lets just log the "message" and not the "stack".  As it is, the logging is too verbose at the moment, and I think the message will be enough.  Thanks.

Reply:
@ear-dev Done,
Please Test